### DISCUSSION TITLE
Update showcases

### DISCUSSION THREAD

**Discussion by sifferhans** [nuxt/nuxt#16241] [General]
I took a look at the [showcase page](https://nuxt.com/showcase) on your homepage, and I saw that a few of the sites listed does not use Nuxt. 
Could it be an idea to go through and update the showcase list, so as to not falsely say something uses Nuxt?

**Here are a few examples.**
- [https://annual.awwwards.com/](https://annual.awwwards.com/)
Uses Webflow

- [https://hacktoberfest.com/](https://hacktoberfest.com/)
Uses Next.js

- [https://netlify.com/technology-partners](https://netlify.com/technology-partners)

- [https://www.meilisearch.com/](https://www.meilisearch.com/)
Uses Next.js

- [https://amplitude.com/pricing](https://amplitude.com/pricing)
Uses Next.js

- [https://search.creativecommons.org/](https://search.creativecommons.org/)
Redirects to [https://wordpress.org/openverse/](https://wordpress.org/openverse/)

- [https://koala.com/](https://koala.com/)
Uses Next.js

- [https://padthaiwok.com/](https://padthaiwok.com/)
Uses Wordpress

- [https://todoist.com/templates](https://todoist.com/templates)
Uses Next.js

**Comment by TheAlexLichter** [comment]
Without checking all, I assume that these sites *used* Nuxt in the past (e.g. Hacktoberfest gets a new site every year). Maybe we could link the web archive link instead of the current page?

**Comment by devbyray** [comment]
Where can I suggest new websites that are built with Nuxt? @TheAlexLichter

**Reply by BlackDahlia313** [reply]
> Where can I suggest new websites that are built with Nuxt? @TheAlexLichter

curious about this too @TheAlexLichter 

I have a pretty big one to suggest

**Reply by cernymatej** [reply]
@BlackDahlia313 feel free to write all of them here
